assets/tuned/daemon/tuned/plugins/plugin_bootloader.py

```diff
@@ -154,6 +164,25 @@ def _grub2_default_env_patch(self):
 			self._cmd.write_to_file(consts.GRUB2_DEFAULT_ENV_FILE, grub2_default_env)
 		return True
 
+	def _grub2_default_env_unpatch(self):
+		grub2_default_env = self._cmd.read_file(consts.GRUB2_DEFAULT_ENV_FILE)
+		if len(grub2_default_env) <= 0:
+			log.info("cannot read '%s'" % consts.GRUB2_DEFAULT_ENV_FILE)
+			return False
+
+		write = False
+		if re.search(r"^GRUB_CMDLINE_LINUX_DEFAULT=\"\$\{GRUB_CMDLINE_LINUX_DEFAULT:\+\$GRUB_CMDLINE_LINUX_DEFAULT \}\\\$" +
+			consts.GRUB2_TUNED_VAR + "\"$", grub2_default_env, flags = re.MULTILINE):
```

Comment:
just a nit, but maybe you can move the regex definition outside of the `if re.search()` call, so that the `if cond` stays in one line, that may make it a bit easier to read (and same at a few other locations in the patch)

Reply:
So, NTO takes the tuned daemon from here: https://github.com/redhat-performance/tuned  as is.  If you feel this should be fixed, please create an issue in that project or I can.  Up to you. :)  But NTO ships the tuned daemon as is and only supplies its own patches in the `assets/tuned/patches` directory as needed.